### PR TITLE
[CO-322] Fix Swagger names in the spec

### PR DESCRIPTION
**Pull request by parsonsmatt**
## Description

The previous swagger spec had names like `WalletResponse<[Wallet]>` where `<[]>` are banned characters. This PR replaces them with URI friendly characters -- specifically, the above would now be `WalletResponse-_Wallet_`.

## Linked issue

https://iohk.myjetbrains.com/youtrack/issue/CO-322

## Type of change

- [x] 🐞 Bug fix (non-breaking change which fixes an issue)
- [~] 🛠 New feature (non-breaking change which adds functionality)
- [~] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)
- [~] 🏭 Refactoring that does not change existing functionality but does improve things like code readability, structure etc
- [x] 🔨 New or improved tests for existing code
- [~] ⛑ git-flow chore (backport, hotfix, etc)

## Developer checklist

- [x] I have read the [style guide](https://github.com/input-output-hk/cardano-sl/blob/develop/docs/style-guide.md) document, and my code follows the code style of this project.
- [x] If my code deals with exceptions, it follows the [guidelines](https://github.com/input-output-hk/cardano-sl/blob/develop/docs/exceptions.md).
- [x] I have updated any documentation accordingly, if needed. Documentation changes can be reflected in opening a PR on [cardanodocs.com](https://github.com/input-output-hk/cardanodocs.com), amending the inline [Haddock](https://www.haskell.org/haddock/) comments, any relevant README file or one of the document listed in the [docs](https://github.com/input-output-hk/cardano-sl/tree/develop/docs) directory.
- [x] CHANGELOG entry has been added and is linked to the correct PR on GitHub.

## Testing checklist

- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: wallet-new/cardano-sl-wallet-new.cabal [hunk 1]
```diff
@@ -617,6 +617,7 @@ test-suite wallet-new-specs
                     , formatting
                     , hedgehog
                     , hspec
+                    , insert-ordered-containers
```
Comment (KtorZ) [inline wallet-new/cardano-sl-wallet-new.cabal]: We don't need this import right?

Comment (KtorZ) [reply wallet-new/cardano-sl-wallet-new.cabal]: I always try to use `Control.Lens.At` instead of bringing in this dep :/ ...

Comment (KtorZ) [reply wallet-new/cardano-sl-wallet-new.cabal]: Or `Control.Lens.Indexed` in that case actually

Comment (parsonsmatt) [reply wallet-new/cardano-sl-wallet-new.cabal]: I needed to get access to the keys of the map and couldn't figure out how to do that with lens :\

**Pull request merged**